libs/utils/trace_analysis.py

```diff
@@ -1303,3 +1303,38 @@ def plotClusterFrequencyResidency(self, clusters=None,
 
         self._plotFrequencyResidency(residencies, 'cluster', xmax, pct, active)
 
+    def getContextSwitchCount(self):
```

Comment:
OK for the first thing.

Regarding the second part, this is meant to be "per-CPU" task context switch count, so I just return that information for all CPUs and then the user can pick the CPU he or she is interested in.